packages/p2p/src/components/my-ads/my-ads-summary.jsx

```diff
@@ -21,7 +27,7 @@ const AdSummary = ({ offer_amount, offer_currency, transaction_currency, price_r
             {!offer_amount ? '...' : ' '}
             {offer_amount && (
                 <span className='p2p-my-ads__form-summary--bold'>
-                    {display_offer_amount} {offer_currency}
+                    {display_offer_amount} {general_store.client.currency}
```

Comment:
Should be, since the order will be completed only if the funds are transferred to the other account, which would be an account of the same currency, right?

Reply:
But okay, I'll change this back to `offer_currency` to keep it foolproof.  👍

Reply:
So, I checked and the prop that is sent for `offer_currency` is `currency` from `Dp2pContext` which now is `general_store.client.currency`.